load.php

```diff
@@ -25,6 +25,11 @@
 define( 'PERFLAB_MODULES_SETTING', 'perflab_modules_settings' );
 define( 'PERFLAB_MODULES_SCREEN', 'perflab-modules' );
 
+// Load Composer dependencies if applicable.
+if ( file_exists( PERFLAB_PLUGIN_DIR_PATH . '/vendor/autoload.php' ) ) {
+	require_once PERFLAB_PLUGIN_DIR_PATH . '/vendor/autoload.php';
+}
+
```

Comment:
Please use manual `require_once` calls for the new PHP files added.

Including an autoloader in a distributed WordPress plugin is not trivial, and if done the regular way (as you're doing here) is at risk of conflicts with other plugins.

I think we should keep it simple and stick with the existing old school pattern of `require_once`ing the files. Changing that would justify its own issue and discussion.

Reply:
@felixarntz this has been addressed as part of the refactoring done against the comment at https://github.com/WordPress/performance/pull/864/#discussion_r1367128918